coursedata/level-defaults/en.yaml

```diff
@@ -167,7 +167,7 @@
 
          ## Example Hedy Code
          ```
-         name is ask 'what's your name?'
```

Comment:
Ah interesting, this code works but not in the right way (it creates an assignment) 
This fix is great but we should look at the root cause too!